src/shared/Atlassian.Bitbucket.Tests/BitbucketAuthenticationTest.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using GitCredentialManager.Tests.Objects;
+using Xunit;
+
+namespace Atlassian.Bitbucket.Tests
+{
+    public class BitbucketAuthenticationTest
+    {
+        [Theory]
+        [InlineData("jsquire", "password")]
+        public async Task BitbucketAuthentication_GetBasicCredentialsAsync_SucceedsAfterUserInput(string username, string password)
+        {
+            var context = new TestCommandContext();
+            context.Terminal.Prompts["Username"] = username;
+            context.Terminal.SecretPrompts["Password"] = password;
+            System.Uri targetUri = null;
+
+            var bitbucketAuthentication = new BitbucketAuthentication(context);
+
+            var result = await bitbucketAuthentication.GetBasicCredentialsAsync(targetUri, username);
+
+            Assert.NotNull(result);
+            Assert.Equal(username, result.Account);
+            Assert.Equal(password, result.Password);
+        }
+
+        [Theory]
+        [InlineData("jsquire", "password")]
+        public async Task BitbucketAuthentication_ShowOAuthRequiredPromptAsync_SucceedsAfterUserInput(string username, string password)
```

Comment:
The inline data is not being used (and triggering a warning in the CI), so let's convert this `Theory` to a `Fact` and drop the unused params.

```suggestion
        [Fact]
        public async Task BitbucketAuthentication_ShowOAuthRequiredPromptAsync_SucceedsAfterUserInput()
```